bazel/scripts/run_integ_tests.sh

```diff
@@ -0,0 +1,239 @@
+    for TARGET in "${EXTENDED_TEST_TARGETS[@]}"
+    do
+        echo "${TARGET}"
+    done
+}
+
+list_traffic_server_tests() {
+    echo "Tests that require the traffic server:"
+    bazel query "attr(tags, traffic_server_test, kind(py_test, //lte/gateway/python/integ_tests/s1aptests/...))"
+}
+
+setup_extended_tests() {
+    echo "Setting up the environment for the extended tests."
+    echo "Building..."
+    bazel build "//lte/gateway/python/integ_tests/s1aptests:test_modify_mme_config_for_sanity" --define=on_magma_test=1
+    echo "Executing..."
+    sudo "${MAGMA_ROOT}/bazel-bin/lte/gateway/python/integ_tests/s1aptests/test_modify_mme_config_for_sanity"
+    echo "Setup finished successfully."
+}
+
+teardown_extended_tests() {
+    echo "Cleaning up the environment after the extended tests."
+    echo "Building..."
+    bazel build "//lte/gateway/python/integ_tests/s1aptests:test_restore_mme_config_after_sanity" --define=on_magma_test=1
+    echo "Executing..."
+    sudo "${MAGMA_ROOT}/bazel-bin/lte/gateway/python/integ_tests/s1aptests/test_restore_mme_config_after_sanity"
+    echo "Cleanup finished successfully."
+}
+
+run_test() {
+    local TARGET=$1
+    local TARGET_PATH=${TARGET%:*}
+    local SHORT_TARGET=${TARGET#*:}
+    (
+        set -x
+        echo "BUILDING TEST: ${TARGET}"
+        bazel build "${TARGET}" --define=on_magma_test=1
+        echo "RUNNING TEST: ${TARGET}"
+        sudo "${MAGMA_ROOT}/bazel-bin/${TARGET_PATH}/${SHORT_TARGET}"
+    )
+}
+
+print_summary() {
+    local NUM_SUCCESS=$1
+    local TOTAL_TESTS=$2
+    echo "SUMMARY: ${NUM_SUCCESS}/${TOTAL_TESTS} tests were successful."
+    for TARGET in "${!TEST_RESULTS[@]}"
+    do
+        echo "  ${TARGET}: ${TEST_RESULTS[${TARGET}]}"
+    done
+}
+
+###############################################################################
+# SCRIPT SECTION
+###############################################################################
+
+TARGET_PATH="${1:-}"
+
+if [[ "${TARGET_PATH}" == "--help" ]];
+then
+    help
+    exit 0
+fi
+
+declare -a EXTENDED_TEST_TARGETS
+declare -A TEST_RESULTS
+NUM_SUCCESS=0
+NUM_RUN=1
+
+cd "${MAGMA_ROOT}"
+
+EXTENDED_TEST_SETUP="//lte/gateway/python/integ_tests/s1aptests:test_modify_mme_config_for_sanity"
+EXTENDED_TEST_TEARDOWN="//lte/gateway/python/integ_tests/s1aptests:test_restore_mme_config_after_sanity"
+SKIP_EXTENDED_SETUP_AND_TEARDOWN="false"
+
+if [[ "${TARGET_PATH}" == "--list" ]];
+then
+    list_all_tests
+    exit 0
+fi
+
+if [[ "${TARGET_PATH}" == "--list-extended" ]];
+then
+    list_extended_tests
+    exit 0
+fi
+
+if [[ "${TARGET_PATH}" == "--list-traffic-server" ]];
+then
+    list_traffic_server_tests
+    exit 0
+fi
+
+if [[ "${TARGET_PATH}" == "--setup-extended" || "${TARGET_PATH}" == "${EXTENDED_TEST_SETUP}" ]];
+then
+    setup_extended_tests
+    exit 0
+fi
+
+if [[ "${TARGET_PATH}" == "--teardown-extended" || "${TARGET_PATH}" == "${EXTENDED_TEST_TEARDOWN}" ]];
+then
+    teardown_extended_tests
```

Comment:
It would be great if running `$MAGMA_ROOT/bazel/scripts/run_integ_tests.sh //lte/gateway/python/integ_tests/s1aptests:test_restore_mme_config_after_sanity` would not fail if there is nothing to restore. Perhaps we can check for the existence of the config files? If there are no config files we simply skip?

Reply:
Fixed.